.github/workflows/qe-image.yml

```diff
@@ -0,0 +1,52 @@
+name: Push qe images for new tag
+on:
+  push:
+    tags:
+      - '*'
+env:
+  IMAGE_REGISTRY: quay.io
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
```

Comment:
As it is only 1 value on the os value of the matrix, you should move the value here (ubuntu-24.04), check comment about versioning

Reply:
Updated